test/unit/spec/v2/view-builder/utils/LinksUtil_spec.js

```diff
@@ -4,11 +4,15 @@ import { getSwitchAuthenticatorLink, getFactorPageCustomLink } from 'v2/view-bui
 import Settings from '../../../../../../src/models/Settings';
 
 describe('v2/utils/LinksUtil', function() {
-  const mockAppState = (remediationFormName, hasMoreThanOneAuthenticator, isPasswordRecovery) => {
+  const mockAppState = (remediationFormName, hasMoreThanOneAuthenticator, isPasswordRecovery, hasOneAuthenticator) => {
```

Comment:
can we change this to using an object? Now we have enough arguments to do something like:
```
const mockAppState = (remediationFormName, options) => {
...
if (formName === options.remediationFormName) {
        if (options.authenticatorCount === 'many') {
          return [ { label: 'some authenticator '}, { label: 'another authenticator' } ];
        } else if (options.authenticatorCount === 'one') {
          return [ { label: 'some authenticator '} ];
        }
      }

```

This way it is easier to read the tests, easier to skip arguments, and easier to add more arguments in the future
The test would be something like
```
const appState = mockAppState(FORMS.SELECT_AUTHENTICATOR_ENROLL, { authenticatorCount: 'one' });
```

Reply:
this looks much better. thanks. have updated the PR.